server.js

```diff
@@ -4197,6 +4197,54 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Twitter follow badge.
+camp.route(/^\/twitter\/follow\/@?([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var user = match[1]; // eg, shields_io
+  var format = match[2];
+  var options = {
+    url: 'http://cdn.syndication.twimg.com/widgets/followbutton/info.json?screen_names=' + user
+  };
+  var badgeData = getBadgeData('Follow', data);
+  badgeData.text[0] = 'Follow ' + user;
+  badgeData.colorscheme = '55ACEE';
+  badgeData.logo = badgeData.logo || logos.twitter;
+  badgeData.links = [
+    'https://twitter.com/intent/follow?screen_name=' + user
+  ];
+  if (data.nobubble) {
+    badgeData.nobubble = data.nobubble;
+    sendBadge(format, badgeData);
+    return;
+  }
+  if (badgeData.template === 'social') {
```

Comment:
Could we have a call to twitter's api in the general case? We can set `badgeData.text[1]` to the empty string if it fails, and show only the left-hand-side.